client/src/components/session-decrypt/decrypt-input.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC, useState, useEffect } from 'react';
+import { Card, CardContent, Divider, Grid, Stack, Typography } from '@mui/material';
+import { Form, Formik, useFormikContext } from 'formik';
+import * as Yup from 'yup';
+import { LockTwoTone } from '@mui/icons-material';
+import { PasswordInput } from '@components/forms/password-input';
+import { CustomFile, FileUpload } from '@components/file-upload/file-upload';
+
+const validationSchema = Yup.object().shape({
+  privateKey: Yup.string().uuid('Invalid Private Key').required('Please input your Private Key.')
+});
+
+export interface CompanyInputFormProps {
+  onFileUpload: (file: CustomFile) => void;
+  setPrivateKey: (privateKey: string) => void;
+}
+
+interface valueProps {
+  privateKey: string;
+}
+
+export const DecryptInputForm: FC<CompanyInputFormProps> = (props) => {
+  const [initialValues, setInitialValues] = useState({
+    privateKey: ''
+  });
```

Comment:
You don't really need this, this would be a way to handle values that change from first render.

Since we are asking for private key, we only expecting it to change when the user providers info and does not change the initialValues value. So this can be removed.

On formik, you can just change initialValues to { privateKey: '' }